frontend/lib/i18n/locales/nl/data_entry.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Invoeren",
+  "entry_saved": "Je invoer is opgeslagen",
+  "abort": {
+    "action": "Invoer afbreken",
+    "title": "Wat wil je doen met je invoer?",
+    "description": "<p>Ga je op een later moment verder met het invoeren van dit stembureau? Dan kan je de invoer die je al hebt gedaan bewaren.</p><p>Twijfel je? Overleg dan met de coördinator.</p>",
+    "save_input": "Invoer bewaren",
+    "discard_input": "Niet bewaren"
+  },
+  "pick_polling_sation": "Kies een stembureau",
+  "success": {
+    "return_paper": "Geef het papieren proces-verbaal terug aan de coördinator.",
+    "second_entry_info": "Een andere invoerder doet straks de tweede invoer."
+  },
+  "completed": {
+    "all_entries_completed": "Alle stembureaus zijn ingevoerd",
+    "thank_you": "Bedankt voor je hulp!",
+    "info": "De resultaten van alle stembureaus kunnen nu opgeteld gaan worden om de uitslag van de {election.name} vast te stellen.",
+    "wait_for_instructions": "Wacht op instructies van de coördinator."
+  },
+  "list": {
+    "not-found": "Geen lijst gevonden voor {listNumber}"
```

Comment:
There's a dash here instead of an underscore. Same is the case in `messages.json`.

Reply:
changed to underscores